frontend/src/ol/staticLayers/vectorSourceLoader.js

```diff
@@ -18,15 +18,16 @@ const vectorSourceLoader = (layerData, handleIsLoading, title, type) => {
             let attributes = [];
 
             if (layerData.attribute) {
+              const escapedAttribute = encodeURIComponent(layerData.attribute);
```

Comment:
This starts failing with "APIError: Parameter 'attributeId' must be url encoded. Its value may not contain reserved characters.